app/models/legal_aid_application.rb

```diff
@@ -505,7 +505,8 @@ def hmrc_response_use_case_one
   end
 
   def uploading_bank_statements?
-    provider.bank_statement_upload_permissions? && (!provider_received_citizen_consent? || attachments.bank_statement_evidence.exists?)
+    citizen_consent_given = provider_received_citizen_consent.nil? ? true : provider_received_citizen_consent?
```

Comment:
instead of coercing `nil` into `true` could we put a check for `legal_aid_application.passported?` at the point that this was breaking before this change was added?

Reply:
@willc-work can we get this in and i do  a separate PR - UAT done now and then you can rebase your CFE change branch on main